package.json

```diff
@@ -18,6 +18,7 @@
   },
   "license": "MIT",
   "main": "dist/htmlhint.js",
+  "types": "dist/core/core.d.ts",
```

Comment:
I'm not that familiar with adding the types to NPM, but this file doesn't exist in the dist folder currently

Reply:
I added the files to the repository, but I don't think we need that in this repository because the `dist` folder is autogenerated.